apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -68,6 +210,16 @@ export async function aiGrade({
   });
 
   serverJob.executeInBackground(async (job) => {
+    job.info('Checking for embeddings for all submissions.');
+    const emb_count = await getAllSubmissionEmbeddings({
```

Comment:
A function named "getThings" implies that "things" will be returned. Consider `generateSubmissionEmbeddings` or the like.

This isn't C, no need to use abbreviations like `emb_count`. Consider `newEmbeddingsCount` instead.

Reply:
done